homeassistant/components/thethingsnetwork/const.py

```diff
@@ -0,0 +1,15 @@
+"""The Things Network's integration constants."""
+
+from homeassistant.const import Platform
+
+DOMAIN = "thethingsnetwork"
+TTN_API_HOSTNAME = "eu1.cloud.thethings.network"
+
+PLATFORMS = [Platform.SENSOR]
+
+CONF_HOSTNAME = "hostname"
+CONF_API_KEY = "access_key"
+CONF_APP_ID = "app_id"
```

Comment:
Oh right. Can we use CONF_HOST and CONF_API_KEY from homeassistant.core instead?